docs/tutorial/part-seven/index.md

```diff
@@ -177,23 +177,20 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
 
 // highlight-start
 exports.createPages = ({ graphql, actions }) => {
-  return new Promise((resolve, reject) => {
-    graphql(`
-      {
-        allMarkdownRemark {
-          edges {
-            node {
-              fields {
-                slug
-              }
+  return graphql(`
```

Comment:
After talking to @pieh, I think we are on board with this. I agree that it is kinda weird to show returning a new Promise when we already have a promise returned from the graphql call. We also think that we want to keep these examples as explicit as possible, while balancing the "copy-pastability" of the examples, i.e. keeping them as real world as possible.

As such, let's maybe just add a comment before the graphql invocation that illustrates that the graphql call returns a Promise? e.g.

```suggestion
  // the graphql function call returns a Promise
  // see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise for more info
  return graphql(`
```